docs/class5/connexml.md

```diff
@@ -328,11 +328,13 @@ An existing call is transferred to a different destination using the `Dial` ver
 |`timeout`|timeout in <Dial> lets you specify the maximum waiting time in seconds for the dialed party to answer|||
 
 |**Noun**|**Description**|
-|-------------|----------|
+|--------|---------------|
 |`Number`|Its is an E.164 phone number|
 |`Queue`|Its a queue name|
 |`Client`|It specifies a client identifier to dial|
 |`Conference`|You can connect to a conference room using the `Dial` verb's `Conference` noun|
+|`Voicemail`| Your voicemails can be played using the `Voicemail` noun. [Click here](https://docs.connexcs.com/class5/voicemail/#voicemail-dialpad-options) to know more options for the Voicemail Dialpad options|
```

Comment:
You can access Voicemail using the `Dial` verb's `Voicemail` noun.

Reply:
done